### DISCUSSION TITLE
Can not execute a binary programme from a shared directory

### DISCUSSION THREAD

**Discussion by kobecow** [utmapp/UTM#6135] [Linux]
**Describe the issue**  
I couldn't run Haskell binary program in a shared directory. Error message was `Segmentation fault (core dumped)`. I could run the binary program in a different directory. That means the program doesn't contain bug to run.
I created a shared dir with virtFS documented in the [docs](https://docs.getutm.app/guest-support/linux/#virtfs).

**Configuration**  
* UTM Version:  4.4.5 
* macOS Version: Sonoma 14.2.1 (23C71)
* Mac Chip (Intel, M1, ...):  M3 Pro
* GuestOS Ubuntu 22.04.3 LTS

**Comment by michaelsargent** [comment]
This looks to be a virtFS issue and not UTM.  Perhaps you can run network file sharing like cifs?
